### PR TITLE
Add PHP Core Roundup 20

### PR DESCRIPTION
**Pull request by Ayesh**

### CODE REVIEW

**Comment by pronskiy** [comment]
@Ayesh, great post! 
Since this one is devoted to the PHP 8.4 release, how about copying/including short paragraphs about Hooks, aviz, HTML%, and BCMath? I know you've already covered them previously, but it's be unfortunate to miss them in this post. What do you think?

File: source/_posts/2024-11-21-php-core-roundup-20-php84.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+---
+
+## Support PHP Foundation
+
+At The PHP Foundation, we support, promote, and advance the PHP language. We financially support ten PHP core developers to contribute to the PHP project. You can help support PHP Foundation on [OpenCollective](https://opencollective.com/phpfoundation) or via [GitHub Sponsors](https://github.com/sponsors/ThePHPF).
+
+A big thanks to all our sponsors — PHP Foundation is all of us!
+
+Follow us on Twitter/X [@ThePHPF](https://twitter.com/thephpf) or  [Mastodon](https://phpc.social/@thephpf) to get the latest updates from the Foundation.
```
Comment (pronskiy) [inline source/_posts/2024-11-21-php-core-roundup-20-php84.md:109]:
```suggestion
Follow us on Twitter/X [@ThePHPF](https://twitter.com/thephpf),  [Mastodon](https://phpc.social/@thephpf), [LinkedIn](https://www.linkedin.com/company/phpfoundation/), and [Bluesky](https://bsky.app/profile/thephpf.bsky.social) to get the latest updates from the Foundation.
```

**Review by pronskiy — CHANGES_REQUESTED** [review]

**Comment by Ayesh** [comment]
I added a somewhat minimal copy-pasta from the previous post about aviz and hooks, bcmath, and dom.

**Comment by Ayesh** [comment]
I'll also add Lazy objects, please do not merge yet :)

**Comment by Ayesh** [comment]
Ready for review :)

I tried to keep the post minimal to avoid duplication, and linked to all available doc pages on php.net now.

I will keep an eye and update links. php/doc-en#3872) tracks them.

File: source/_posts/2024-11-21-php-core-roundup-20-php84.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  - roundup
+author:
+  - name: Ayesh Karunaratne
+    url: https://aye.sh
+
+  - name: Sergey Panteleev
+    url: https://sergeypanteleev.com
+---
+
+We are thrilled to announce that after a year of hard work, dedication, and collaboration, **PHP 8.4** is officially here!
```
Comment (pronskiy) [inline source/_posts/2024-11-21-php-core-roundup-20-php84.md:15]:
```suggestion
We are thrilled to announce that after a year of hard work, dedication, and collaboration, [**PHP 8.4**](https://www.php.net/releases/8.4/) is officially here!
```

**Pull request merged**